packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -0,0 +1,279 @@
+      photo,
+      postMenstrualAge,
+      postMenstrualAgeLabel,
+      preferredFirstName,
+      ...customProps
+    } = this.props;
+
+    const mainClasses = classNames(
+      'terra-DemographicsBanner',
+      { 'terra-DemographicsBanner--deceased': deceasedDate },
+      customProps.className,
+    );
+
+    delete customProps.className;
+
+    return (
+      <section className={mainClasses} {...customProps}>
+        <div className="terra-DemographicsBanner-profile-photo">
+          {this.props.photo}
+        </div>
+        <div className="terra-DemographicsBanner-content">
+          <div className="terra-DemographicsBanner-row">
+            <h1 className="terra-DemographicsBanner-person-name">
+              { personName }
+              { preferredFirstName && <span className="terra-DemographicsBanner-preferred-first-name">
+                { preferredFirstName }
+              </span> }
+            </h1>
+            <div className="terra-DemographicsBanner-application-content">
+              {applicationContent}
+            </div>
+          </div>
+          <div className="terra-DemographicsBanner-row">
+            <div className="terra-DemographicsBanner-person-details">
+              {this.personDetails()}
+            </div>
+            <div className="terra-DemographicsBanner-identifiers">
+              {this.applicationIdentifiers()}
+            </div>
+          </div>
+        </div>
+      </section>
+    );
+  }
+
+  renderSmallDemographicsBanner() {
+    const {
+      age,
+      applicationContent,
+      dateOfBirth,
+      deceasedDate,
+      deceasedDateLabel,
+      gender,
+      gestationalAge,
+      gestationalAgeLabel,
+      identifiers,
+      personName,
+      photo,
+      postMenstrualAge,
+      postMenstrualAgeLabel,
+      preferredFirstName,
+      ...customProps
+    } = this.props;
+
+    const mainClasses = classNames(
+      'terra-DemographicsBanner',
+      { 'terra-DemographicsBanner--deceased': deceasedDate },
+      customProps.className,
+    );
+
+    delete customProps.className;
+
+    return (
+      <section className={mainClasses} {...customProps}>
+        <h1 className="terra-DemographicsBanner-person-name">
+          <span>
+            { personName }
+            { preferredFirstName && <span className="terra-DemographicsBanner-preferred-first-name">
+              { preferredFirstName }
+            </span> }
+          </span>
+        </h1>
+        <div className="terra-DemographicsBanner-person-details">
+          {this.personDetails()}
+          {this.applicationIdentifiers()}
+        </div>
+        <div className="terra-DemographicsBanner-application-content">
+          {applicationContent}
+        </div>
+      </section>
+    );
+  }
+
+  render() {
+    return (
+      <ResponsiveElement
+        responsiveTo="window"
+        defaultElement={this.renderSmallDemographicsBanner()}
+        tiny={this.renderSmallDemographicsBanner()}
```

Comment:
That works. https://github.com/cerner/terra-core/pull/280/commits/9c6700b72e02263f566a60399a2899e4acd4643f